Steepfile

```diff
@@ -0,0 +1,8 @@
+D = Steep::Diagnostic
+
+target :lib do
+  signature "sig"
+  check "lib" # Directory name to check
+
+  configure_code_diagnostics(D::Ruby.silent)  # `silent` diagnostics setting
```

Comment:
Why do we need this?

Reply:
OK. Could you share the documentation for this configuration?

Reply:
I refer to this document.
https://github.com/soutaro/steep/blob/master/manual/ruby-diagnostics.md

Reply:
Thanks.

Based on it, how about using `lenient` instead of `silent`?

It seems that `silent` reports nothing even for a RBS syntax error: https://github.com/soutaro/steep/blob/master/manual/ruby-diagnostics.md#rubyannotationsyntaxerror
Is it useful for us?

It seems that `lenient` reports an error only for a RBS syntax error. It reports something or nothing for other cases.
It will be useful for us. For example, we may be able to check our RBS in CI.

Reply:
Thank you for your advice and I agree with that.
I updated with this commit [e615a4b](https://github.com/red-data-tools/red-datasets/pull/238/commits/e615a4bf051012c9445313863c950ddbe025ff0c).